frontend/containers/User.js

```diff
@@ -46,21 +54,11 @@ class User extends Component {
               </div>
             </div>
           </div>
-          <UserPreferences email={this.getEmail()} />
+          <UserPreferences />
```

Comment:
Should we keep passing in the email, so we don't have to make this request twice? It isn't a big deal, more curious what your thoughts are about it.

Reply:
This isn't worth blocking merging this on.

Reply:
It's been some time since I made this change but iirc, I tried to pass it through and it didn't work since the 2 were being loaded at the same time and I'd get a user loaded but with no preferences. It's probably feasible but my frontend knowledge isn't high and I couldn't figure it out